metadata_service/proxy/atlas_proxy.py

```diff
@@ -372,6 +377,24 @@ def _get_reports(self, guids: List[str]) -> List[ResourceReport]:
 
         return parsed_reports
 
+    def _get_owners(self, data_owner: list, fallback_owner: str) -> List[User]:
```

Comment:
if data_owner is a list maybe it should be data_owners ?

Reply:
DONE